formidable/views.py

```diff
@@ -21,6 +24,86 @@
 from formidable.serializers.presets import PresetsSerializer
 
 
+logger = logging.getLogger(__name__)
+
+
+def extract_function(func_name):
```

Comment:
indead of using this, the best thing to do is to use the import module writing by django rest framework.